### ISSUE TITLE
DT-1214: Remove support for Features in BLT 12

### ISSUE THREAD

**Issue by danepowell** [acquia/blt#4040]
Features hasn't had a new release in years, and the project homepage itself advocates never using Features in Drupal 8. More importantly, it's not compatible with D9 at all.

Is anyone actually using it with BLT 11?

It seems like anything that can be done with Features can be done better with Config Split and similar tools, and we should remove support in BLT 12 / D9.

**Comment by leymannx** [comment]
I don't have much experience with BLT yet. But Config Split is not replacement for Features considering the following use case:

Big multi-site project. Some of the multi-site instances sharing some content types and/or image styles, text formats etc. To sync these content types and/or image styles, text formats etc. across all multi-site instances Features comes into play.

Let's say this shared content type now should get a new field. Add this field on one instance, export the Features module that holds this content type, then import it in all others multi-site instances, and finally export config on all sites.

As long as BLT supports multi-sites it's probably worth keeping the support for Features? Under the condition that Features gets ready for Drupal 9 of course.

**Comment by danepowell** [comment]
@mikemadison13 thoughts on this?

**Comment by mikemadison13** [comment]
So, it seems to me that "Features" as a module isn't going to be a thing but a "features-like" approach to modular config will likely continue to be a thing. Personally, I think that you can do the modular approach w/o using Features itself (and will be forced to do so in D9). My thoughts are that if Features doesn't work with D9, then as of BLT 12 we don't support it. Period. Having said that, I think the use case that @leymannx  is describing here continues to be widely used (even if not using Features itself).  I would assume that setting BLT to a `config-strategy: none` would allow for it.

**Comment by dpagini** [comment]
I could "2nd" much of what Mike says. I used features in a few projects for a long time to achieve exactly what @leymannx is suggesting, but I constantly had problems with it. Eventually I made the decision to abandon features (a large task) and implement more of a contrib approach, but our modules just had a lot more config than a traditional module. This meant lots of update hooks when config changed, which was definitely a downside. Overall the amount of issues went way down, and I achieved all of this with the `config-strategy: none` like Mike says as well.
